tests/integration/api/test_eda_credential.py

```diff
@@ -1002,3 +1002,29 @@ def test_custom_credential_type_empty_inputs(
     assert response.status_code == status.HTTP_201_CREATED
     result = response.data
     assert result["name"] == "demo-credential"
+
+
+@pytest.mark.django_db
+def test_copy_eda_credential_success(
+    admin_client: APIClient,
+    default_registry_credential,
+):
+    response = admin_client.post(
+        f"{api_url_v1}/eda-credentials/{default_registry_credential.id}/copy/"
+    )
+    assert response.status_code == status.HTTP_201_CREATED
+    new_credential = response.data
+    assert new_credential["id"] != default_registry_credential.id
+    assert (
+        new_credential["credential_type"]["id"]
+        == default_registry_credential.credential_type.id
+    )
+    assert default_registry_credential.name in new_credential["name"]
```

Comment:
can you add a check to ensure the encrypted field is decryptable and its value is the same as the original credential?